app/src/services/course.services.ts

```diff
@@ -8,29 +8,46 @@ export interface CourseInterface {
   title: string;
   category: string;
   level: string;
-  time: number;
   description: string;
+  estimatedHours: number;
 }
 
 /**
  * IN ALL METHODS THE TOKEN HAS BEEN COMMENTED OUT, SINCE WE DON'T HAVE A TOKEN YET
  */
 
 // Create a new course
-const createCourse = async ({ title, category, level, time, description }: CourseInterface, token: string) => {
+const createCourse = async ({ title, category, level, estimatedHours, description }: CourseInterface, token: string) => {
+  console.log("Creating course with title: " + title);
   return await axios.post(
     `${BACKEND_URL}/api/courses`,
     {
       title: title,
       description: description,
-    }/*,
-    { headers: { Authorization: `Bearer ${token}` } }*/
+      category: category,
+      level: level,
+      estimatedHours: estimatedHours,
+    },
+    { headers: { Authorization: `Bearer ${token}` } }
   );
 };
 
+// Create a new section for a course FIXME: should this be in section.services ??
+const updateCoverImage = async ( id: any, token: string) => {
+ 
+  return await axios.post(
+    `${BACKEND_URL}/api/course/update/coverImage/${id}`,
```

Comment:
Do not have actions in the endpoint. Also endpoints should be pluralized (courses instead of course)

This would be api/courses/coverImage/:id

I also believe, if I'm not mistaken, that we should not use camelcase in endpoints. @Jatewo @Havkost, do you guys remember?

Reply:
Also, should we use PATCH instead of POST?